src/Connectivity/Connectivity.jl

```diff
@@ -0,0 +1,646 @@
+module Connectivity
```

Comment:
I would add a docstring to the module itself to document what it regroups

Reply:
@kshyatt and I will be doing a doc overhaul prior to 2.0.

Reply:
Still I guess we can add something

Reply:
I'd really rather do it consistently once we've made decisions as to where things are going to go. I anticipate some movement of functions among different modules.

Reply:
I'll open an issue to make sure we get to this, though. Thanks for bringing it up.

Reply:
Ref #1376